launchable/test_runners/pytest.py

```diff
@@ -84,22 +94,24 @@ def _path_to_class_name(path):
 def _pytest_formatter(test_path):
     for path in test_path:
         t = path['type']
+        n = path['name']
```

Comment:
Could you change to use `.get()` method like this PR https://github.com/launchableinc/cli/pull/378?